### DISCUSSION TITLE
show in maps

### DISCUSSION THREAD

**Discussion by fjfuente13** [icepick3000/AirtagAlex#6] [Q&A]
hi Alex, I think it is awesome

but I would like to view de .csv points in a map.

can you think that's its easy?

tk in advance

**Comment by ak2k** [comment]
I think it would be a great feature to show location history on Google Maps. I haven't looked into the easiest way to do this integration, but I recall that Google Maps allows you to show points from an imported KML file. Some of the run trackers generate KML files for this purpose. Perhaps converting the CSV to KML is all that's needed?
